backend/sirius-web-spring-collaborative/src/main/java/org/eclipse/sirius/web/spring/collaborative/api/IRepresentationRefreshPolicy.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.spring.collaborative.api;
+
+/**
+ * The interface used to describe the refresh policy of a representation regarding the given change description.
+ *
+ * @author gcoutable
+ */
+@FunctionalInterface
```

Comment:
No need for that annotation but could you document this new concept in the CHANGELOG.adoc please?

Reply:
Done